src/App.js

```diff
@@ -0,0 +1,250 @@
+import Dell from './assets/images/del.jpg';
+import Hp from './assets/images/hp.jpg';
+import Acer from './assets/images/acer.webp';
+import Benq from './assets/images/benq.webp';
+import Lg from './assets/images/lg.jpg';
+
+const itemList = [
+    {
+        img: Dell,
+        name: 'Dell-1',
+        price: '$20',
+        id: 0,
+        count: 0,
+    },
+    {
+        img: Dell,
+        name: 'Dell-2',
+        price: '$50',
+        id: 1,
+        count: 0,
+    },
+    {
+        img: Dell,
+        name: 'Dell-3',
+        price: '$100',
+        id: 2,
+        count: 0,
+    },
+    {
+        img: Hp,
+        name: 'Hp-1',
+        price: '$37',
+        id: 3,
+        count: 0,
+    },
+    {
+        img: Hp,
+        name: 'Hp-2',
+        price: '$68',
+        id: 4,
+        count: 0,
+    },
+    {
+        img: Acer,
+        name: 'Acer-1',
+        price: '$200',
+        id: 5,
+        count: 0,
+    },
+    {
+        img: Acer,
+        name: 'Acer-2',
+        price: '$400',
+        id: 6,
+        count: 0,
+    },
+    {
+        img: Benq,
+        name: 'BenQ-1',
+        price: '$1000',
+        id: 7,
+        count: 0,
+    },
+    {
+        img: Benq,
+        name: 'BenQ-2',
+        price: '$600',
+        id: 8,
+        count: 0,
+    },
+    {
+        img: Lg,
+        name: 'LG-1',
+        price: '$200',
+        id: 9,
+        count: 0,
+    },
+    {
+        img: Lg,
+        name: 'LG-2',
+        price: '$300',
+        id: 10,
+        count: 0,
+    },
+];
+
+// Seting up Local Storage
+let storageItems;
+let storageCounterItem;
+let storageModalItems = [];
+
+if (localStorage.getItem('localStorageList')) {
+    storageItems = JSON.parse(localStorage.getItem('localStorageList'));
+
+    storageModalItems = JSON.parse(
+        localStorage.getItem('localStorageModalItems')
+    );
+
+    storageCounterItem = localStorage.getItem('localStorageCounter');
+} else {
+    localStorage.setItem('localStorageList', JSON.stringify(itemList));
+    storageItems = JSON.parse(localStorage.getItem('localStorageList'));
+
+    localStorage.setItem('localStorageCounter', 0);
+    storageCounterItem = 0;
+
+    localStorage.setItem('localStorageModalItems', JSON.stringify([]));
+}
+
+function App() {
+    const [items, setItems] = useState(storageItems);
+    const [counter, setCounter] = useState(+storageCounterItem);
+    const [modalItems, setModalItems] = useState(storageModalItems);
+    let newItems = [];
+
+    function storage(items) {
+        localStorage.setItem('localStorageModalItems', JSON.stringify(items));
+
+        storageModalItems = JSON.parse(
+            localStorage.getItem('localStorageModalItems')
+        );
+    }
+
+    function counterStorage(count) {
+        localStorage.setItem('localStorageCounter', +count);
+    }
+
+    function addCardToCartHandler(id) {
+        newItems = [...modalItems];
+        let index = [newItems.length];
```

Comment:
index je ocigledno `const` a ne `let` jer nigde nemas redekleraciju tj. `index = `. 
Takodje nije mi bas jasno zasto bi index bio array of length - mogao je biti samo length?!